Standards/scs-0100-v2-flavor-naming.md

```diff
@@ -68,20 +68,28 @@ encoding all details) as well as very detailed longer names.
 
 ## Complete Proposal for systematic flavor naming
 
-| Prefix | CPU | Suffix       | RAM[GiB]   | optional: Disk[GB] | optional: Disk type | optional: extra features                            |
-| ------ | --- | ------------ | ---------- | ------------------ | ------------------- | --------------------------------------------------- |
-| `SCS-` | `N` | `L/V/T/C[i]` | `-N[u][o]` | `[-[Mx]N]`         | `[n/s/l/p]`         | `[_hyp][_hwv][_[arch[N][h][_[G/g]X[N][-M[h]]][_ib]` |
+| Prefix | CPUs & Suffix    | RAM[GiB]        | opt: Disk[GB]&type      | opt: extensions |
+| ------ | ---------------- | --------------- | ----------------------- | ----------------|
+| `SCS-` | N`L/V/T/C`[`i`]` | `-`N[`u`][`o`]` | [`-`[M`x`]N[`n/s/l/p`]] | [`_`EXT]        |
```

Comment:
The backticks are slightly broken, is this intended?

Reply:
Actually, this was intentional. The things that get into the string literally are supposed to be within backticks (i.e. the `SCS-` prefix or the letters indicating the vCPU type), but no longer the brackets [] that indicate something is optional nor the N or M letters that are placeholders for numbers.